docs/resources/logpush_job.md

```diff
@@ -125,13 +125,32 @@ resource "cloudflare_logpush_job" "example_job" {
 - `max_upload_interval_seconds` (Number) The maximum interval in seconds for log batches. Value must be between 30 and 300.
 - `max_upload_records` (Number) The maximum number of log lines per batch. Value must be between 1000 and 1,000,000.
 - `name` (String) The name of the logpush job to create.
+- `output_options` (Block List, Max: 1) (see [below for nested schema](#nestedblock--output_options))
 - `ownership_challenge` (String) Ownership challenge token to prove destination ownership, required when destination is Amazon S3, Google Cloud Storage, Microsoft Azure or Sumo Logic. See [Developer documentation](https://developers.cloudflare.com/logs/logpush/logpush-configuration-api/understanding-logpush-api/#usage).
 - `zone_id` (String) The zone identifier to target for the resource. Must provide only one of `account_id`, `zone_id`.
 
 ### Read-Only
 
 - `id` (String) The ID of this resource.
 
+<a id="nestedblock--output_options"></a>
+### Nested Schema for `output_options`
+
+Optional:
+
+- `batch_prefix` (String)
```

Comment:
can you please add `Description` values to the schema attributes to populate this correctly?

Reply:
Addressed with f27f6c3d563adbf7ffed82c304cc4a913f3dd165.